en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -69,6 +69,18 @@ flow.statistics.enable=true
 stat.tracer.collect_payloads=true
 stat.tracer.collect_mediation_properties=true
 ```
+### Configuring the Micro Integrator to Publish Data to Analytics Server
+Analytics publishing can be configured in `<MI_HOME>/conf/deployment.toml` in '[monitoring]' section.
+```
```

Comment:
shall we add a tip here saying that the following is the default configuration available internally and user will have to add and modify it, if they want to change it only.